src/tablestore.rs

```diff
@@ -55,41 +56,52 @@ impl ReadOnlyBlob for ReadOnlyObject {
     }
 
     async fn read_range(&self, range: Range<usize>) -> Result<Bytes, SlateDBError> {
-        self.object_store
-            .get_range(&self.path, range)
+        let opts = CacheableGetOptions {
+            range: Some(GetRange::Bounded(range)),
+            ..Default::default()
```

Comment:
🤔 I think this is using the default cache rank. I don't think we want that, right? Shouldn't the reader decide the rank? OTOH, not sure I want to pollute `read_range` with the cache rank param....